yarn-project/circuit-types/src/stats/metrics.ts

```diff
@@ -122,6 +122,48 @@ export const Metrics = [
     description: 'Time for the sequencer to execute public function calls for txs in a block.',
     events: ['l2-block-built'],
   },
+  {
+    name: 'setup_kernel_simulation_avg_ms',
+    groupBy: 'block-size',
```

Comment:
What do you think about having a generic `public_kernel_simulation_ms` metric groupings by `kernel_type` (i.e. `setup`, `app`, `teardown`, `tail`). I don't see what benefit we're getting from separating these stats by block size, maybe I'm missing something.

Reply:
Nope, you're missing nothing. I agree.